lib/vquic/msh3.c

```diff
@@ -569,15 +612,17 @@ CURLcode Curl_cf_msh3_create(struct Curl_cfilter **pcf,
   struct Curl_cfilter *cf = NULL;
   CURLcode result;
 
-  (void)data;
   (void)conn;
-  (void)ai; /* TODO: msh3 resolves itself? */
+  infof(data, "msh3: initializing");
+
   ctx = calloc(sizeof(*ctx), 1);
   if(!ctx) {
     result = CURLE_OUT_OF_MEMORY;
     goto out;
   }
 
+  ctx->ai = ai;
```

Comment:
@icing why isn't this passed into the `cf_msh3_connect` function, so I don't have to cache it here?